core-bundle/src/Resources/config/listener.yml

```diff
@@ -55,8 +55,8 @@ services:
             - '%contao.csrf_cookie_prefix%'
         tags:
             # The priority must be higher than the one of the Symfony route listener (defaults to 32)
-            - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest, priority: 36 }
-            - { name: kernel.event_listener, event: kernel.response, method: onKernelResponse }
+            - { name: kernel.event_listener, event: kernel.request, priority: 36 }
+            - { name: kernel.event_listener, event: kernel.response }
```

Comment:
This for example, why is `event: kernel.response` still required here? There's no priority attribute.

Reply:
Because otherwise Symfony throws an exception: `Service "contao.listener.csrf_token_cookie" must define the "event" attribute on "kernel.event_listener" tags.` 🤷‍♂

Reply:
Okay, but why? 😄

Reply:
I don't know. But three remaining `event: ` keys should not prevent us from merging this PR.

Reply:
I'm reviewing here and I would like to understand why it doesn't work so that we can either fix it or know why we have inconsistency now. Can you check which condition of `RegisterListenersPass::getEventFromTypeDeclaration()` is failing?

Reply:
I have found the issue. The pass is looking for an `__invoke()` method and if there is none, you either have to specify the `event` or `method` key. If no `method` key is given, it is computed from the `event` key.

I guess it is more correct to specify the method names in our three cases (see 4a95e51b20199174bf7937c100bac534018f9265).

Reply:
> I guess it is more correct to specify the method names in our three cases (see [4a95e51](https://github.com/contao/contao/commit/4a95e51b20199174bf7937c100bac534018f9265)).

I agree with that 👍